History.md

```diff
@@ -3,6 +3,20 @@
 ### Changes
 
 * The `meteor-babel` npm package has been updated to version 7.7.4.
+* `Meteor.user()`, `Meteor.findUserByEmail()` and `Meteor.findUserByUserName()` can take a new
```

Comment:
nitpick: can you add a blank line between the bullet points, similar to the rest of the changelog.

Reply:
Done, well spotted, sorry!  Plus merged 1.8.3 changes to History.md